서영/Week2_중복된 숫자 개수.js

```diff
@@ -0,0 +1,9 @@
+function solution(array, n) {
+    var answer = 0;
```

Comment:
해당 부분도 var 대신 let을 사용하는게 맞겠죠?
```suggestion
    let answer = 0;
```